classes/01class/exercises/c01-e03/Faria-Marcio/c01-e03.txt

```diff
@@ -0,0 +1,15 @@
+PUBLIC IP ADDRESS EC2:
+
+1- On EC2 services I clicked on Launch Instance button;
+2- Chose the default Amazon Machine Image (AMI), which was Amazon Linux;
+3- Chose t2.micro instance type;
+4- Configured the instance to Auto-assign Public IP, to access through SSH, and launched it;
+5- Created a new key pair (AWS_-_Devops_KeyPair.pem);
+6- On Ubuntu I accessed the instance through ssh -i "Devops_KeyPair.pem" ec2-user@ec2-13-54-19-240.ap-southeast-2.compute.amazonaws.com;
+
+
+PRIVATE IP ADDRESS EC2:
+
+- Steps 1, 2, 3 of Public IP Address, and launched the EC2 instance;

```

Comment:
This was actually created again a public IP.
For the second one, you should disable the auto-assign and let AWS only create a private IP. Then you will SSH to the first instance and from that, "jump" to the private one.

Reply:
I forgot to mention in the original file that I disabled Auto-assign Public IP option when I created the Private IP Address EC2 instance. Is that ok to use Elastic IP address to access it

Reply:
The private instance can not have any public IP associated with it.
You need to access the private instance using the private IP. 
In order to do that, you'll need to connect to it through the public instance(which can be called jump box or bastion host).